.eslintrc.json

```diff
@@ -17,40 +17,33 @@
     },
     "overrides": [
         {
-            "files": ["./*.js"],
-            "extends": ["./configs/eslint-common-node.json"]
+            "files": ["**/*.js"],
+            "extends": ["./packages/configs/eslint-common-node.json"]
         },
         {
-            "files": "./src/**/*.ts",
-            "extends": ["airbnb-base", "./configs/eslint-common.json"]
+            "files": "./packages/frontend/src/**/*.ts",
+            "extends": ["airbnb-base", "./packages/configs/eslint-common.json"]
         },
         {
-            "files": "./backend/*.ts",
-            "excludedFiles": "*.test.ts",
-            "extends": ["./configs/eslint-common-node.json"],
+            "files": "./packages/backend/src/**/*.ts",
+            "extends": ["./packages/configs/eslint-common-node.json"],
             "rules": {
-                // NOTE: packageDir is relative to the current working directory when executing linter
-                "import/no-extraneous-dependencies": ["error", { "packageDir": "./" }],
-                "node/no-extraneous-import": "off",
```

Comment:
Since there's dedicated `package.json` file for `backend` package there's no need for these rules overrides.